kraken/litmus/common_litmus.py

```diff
@@ -0,0 +1,98 @@
+import kraken.invoke.command as runcommand
+import yaml
+import logging
+import time
+import sys
+
+
+# Install litmus and wait until pod is running
+def install_litmus(version):
+    runcommand.invoke("kubectl apply -f "
+                      "https://litmuschaos.github.io/litmus/litmus-operator-%s.yaml" % version)
+
+    runcommand.invoke("oc wait deploy -n litmus chaos-operator-ce --for=condition=Available --timeout=30s")
+
+
+def perform_experiment(version_string):
+
+    if version_string.startswith("v"):
```

Comment:
why not stick to a version format and fail otherwise? it will give more clarity to have a single version format rather than expect multiple ones.

Reply:
good idea! I had added this incase we wanted to use "latest" as the version. What are your thoughts here? Should I give the option to use latest

Reply:
Using latest could lead to a situation where we test against a major release that render some existing tests invalid due to changes in the CRD schema, for instance. Controlling which version is deployed can also help to reduce false positives by removing litmus code from the potential changes in the test execution.